index.html

```diff
@@ -2,8 +2,23 @@
 <html lang="en">
   <head>
     <title>Parking Mandates Map</title>
+    <style>
+      html,
+      body,
+      #map {
+        height: 100%;
+        width: 100%;
+      }
+    </style>
```

Comment:
It'd be better to start a CSS file. While this is simple code, it's going to be more maintainable to have all styling live in CSS.